files/initializers/jbuilder.rb

```diff
@@ -0,0 +1 @@
+Jbuilder.key_format(camelize: :lower)
```

Comment:
@MProG  add `.editorconfig` to this project or install https://github.com/datarockets/dotfiles